docker/start.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+export JID=${JID:-alertmanager@example.com}
+export PASSWORD=${PASSWORD:-PASSWORD}
+export LISTEN_ADDRESS=${LISTEN_ADDRESS:-0.0.0.0}
+export LISTEN_PORT=${LISTEN_PORT:-9199}
```

Comment:
Wouldn't it make more sense to have reasonable defaults for the listening address/port (port 9199, 127.0.0.1) and just have an "EXPOSE 9199" in the docker file?